Tests/scs-test-runner.py

```diff
@@ -58,17 +59,63 @@ def get_subject_mapping(self, subject):
         mapping.update(self.subjects.get(subject, {}).get('mapping', {}))
         return mapping
 
+    def generate_compliance_check_jobs(self, subjects, scopes):
+        jobs = []
+        for subject in subjects:
+            for scope in scopes:
+                if scope == "scs-compatible-iaas":
+                    jobs.append([scope, subject])
+                if scope == "scs-compatible-kaas":
+                    k8s_setup = self.get_kubernetes_setup(subject)
+                    for k8s_version in k8s_setup["kube_versions"]:
+                        kaas_subject = f"{subject}.{k8s_version}"
+                        jobs.append([scope, kaas_subject, k8s_version, k8s_setup["kube_plugin"]])
+        return jobs
```

Comment:
I think this should be reverted. The clusters are to be provisioned BEFORE running the jobs; the jobs will find them thanks to the variable assignments.

Reply:
I can mv it to a separate loop, but would place it still inside of the tempfile context.
So it can make use of the temporary directory to store the individual kubeconfig files to automatically remove them when the tests are done.

Reply:
Well, we need a separate working directory for each subject. Let's say it's `os.path.join(tdirname, subject)` (in `run`), and then we can put the reports for each subject there as well as the kubeconfig files. This will make the code in `run` a bit more complicated, but for a good reason.

Reply:
I agree with the handling of the separate working directory and will implement this as you suggest.

However, the original function that creates the jobs was as follows:

```
[jobs = [(scope, subject) for scope in scopes for subject in subjects]
```
(see: [scs-test-runner.py#L178](https://github.com/SovereignCloudStack/standards/blob/ced5954957b025318004f6205e218eb22979336a/Tests/scs-test-runner.py#L178C9-L178C78))

I can revert to this function, but keep in mind that the jobs representing kaas subjects consist of three jobs, each representing one of the three k8s versions being tested against.
The list generation from above will not take this into account.

Reply:
I have already changed it

Reply:
I see, but there is still the question of how to generate the jobs for the Kaas checks.
(see my comment above)

If I have a subject with the scope `scs-compatible-kaas` and I want to test against three different k8s versions, I need a conformance test job for each of them.

Reply:
It's all included in my changes